erpnext/projects/doctype/task/task_tree.js

```diff
@@ -45,9 +48,33 @@ frappe.treeview_settings['Task'] = {
 							method: "erpnext.projects.doctype.task.task.add_multiple_tasks",
 							args: {
 								data: d.get_values(),
-								parent: node.data.value
+								parent: node.data.name ? node.data.name : ""
 							},
-							callback: function() { }
+							callback: function() { glob.make_tree(); }
+						});
+					}
+				});
+				d.show();
+			}
+		},
+		{
+			label:__("Rename"),
+			click: function(node) {
+				var d = new frappe.ui.Dialog({
+					'title': __("Rename Subject"),
```

Comment:
We can simply remove the Rename option from tree view. User can change the subject using Edit button. "Rename" button is confusing, as it is actually not renaming.